k8s/network.go

```diff
@@ -198,7 +211,7 @@ func (a *networkImpl) AddNode(cfg node.Config) (node.Node, error) {
 	}
 
 	uri := node.Status.NetworkMembersURI[0]
-	cli := api.NewAPIClient(uri, constants.DefaultPort, constants.APITimeoutDuration)
+	cli := api.NewAPIClient(uri, constants.DefaultAPIPort, constants.APITimeoutDuration)
```

Comment:
Well that's what the Avalanchego operator sets. I think we can assume this the same way as with a common avago node. If you don't provide any env var, then the default will be taken.

We could therefore check if the user provided a custom port, but to be honest, it would not make much sense for the user to do that - it's a pod, not a process, therefore pods don't share their port number space...